.github/workflows/integration.yml

```diff
@@ -197,13 +194,8 @@ jobs:
         working-directory: ./web
         run: yarn run test:ci
 
-      - name: Fix code coverage paths (web)
-        # Due to the way sonarscanner mounts thing when running in docker, we need to change the coverage file.
-        working-directory: ./web/coverage
-        run: sed -i 's/\/home\/runner\/work\/wps\/wps\/web\//\/github\/workspace\//g' lcov.info
-
-      - name: SonarCloud Scan
```

Comment:
We want to keep sonarcloud, just not the code coverage - we can move it to a different job so it happens in parallel though.